distribution/changelog.txt

```diff
@@ -7,6 +7,7 @@
 - Feature: [#19378] Add command to combine CSG1i.DAT and CSG1.DAT.
 - Feature: [objects#226] Port RCT1 Corkscrew Coaster train.
 - Feature: [objects#229] Port RCT1 go karts with helmets.
+- Feature: [#19341] Add "All Scenery" tab to scenery window.
```

Comment:
Noticed right after approving: this line should be moved up.
```suggestion
- Feature: [#19341] Add "All Scenery" tab to scenery window.
- Feature: [#19378] Add command to combine CSG1i.DAT and CSG1.DAT.
- Feature: [objects#226] Port RCT1 Corkscrew Coaster train.
- Feature: [objects#229] Port RCT1 go karts with helmets.
```

Reply:
Is this based on the PR number?

Reply:
Yes, first by category then by issue or PR number.